File: src/main/java/io/github/alexiscomete/lapinousecond/commands/classes/TestCommand.java
package io.github.alexiscomete.lapinousecond.commands.classes;

import io.github.alexiscomete.lapinousecond.commands.CommandWithAccount;
import io.github.alexiscomete.lapinousecond.entity.Player;
import org.javacord.api.event.message.MessageCreateEvent;

public class TestCommand extends CommandWithAccount {

    public TestCommand() {
        super("test", "test", "test");
    }

    @Override
    public void execute(MessageCreateEvent messageCreateEvent, String content, String[] args, Player p) {
        messageCreateEvent.getMessage().reply("Aucun test en cours, revenez plus tard.");
    }
}

File: src/main/java/io/github/alexiscomete/lapinousecond/commands/classes/TestCommand.kt
package io.github.alexiscomete.lapinousecond.commands.classes

import io.github.alexiscomete.lapinousecond.commands.CommandWithAccount
import io.github.alexiscomete.lapinousecond.entity.Player
import org.javacord.api.event.message.MessageCreateEvent

class TestCommand : CommandWithAccount("test", "test", "test") {
    override fun execute(messageCreateEvent: MessageCreateEvent, content: String, args: Array<String>, p: Player) {
        messageCreateEvent.message.reply("Aucun test en cours, revenez plus tard.")
    }
}